homeassistant/components/ihc/ihcdevice.py

```diff
@@ -29,6 +29,7 @@ def __init__(self, ihc_controller, name, ihc_id: int, info: bool,
 
     async def async_added_to_hass(self):
         """Add callback for IHC changes."""
+        await super().async_added_to_hass()
```

Comment:
Why was this added if it doesn't use restore state ?

Reply:
Just as good practice since we're now extending the implementation from the base `Entity` class to hopefully prevent confusion down the road. I can remove it if we don't think it's helpful.

Reply:
I don't think that we should add something that is not going to be used.

Reply:
OK, I can take them out.